seeds/_sector/nft/_schema.yml

```diff
@@ -10,10 +10,16 @@ seeds:
 
   - name: foundation_ethereum_base_trades_seed
     config:
+      tags: dunesql
       column_types:
-        price_raw: decimal(38)
-        platform_fee_amount_raw: decimal(38)
-        royalty_fee_amount_raw: decimal(38)
+        tx_hash: varbinary
```

Comment:
Why additional columns?

Reply:
The columns where already there in the seed, but these were set as varchar by the automatic type inference, hence the new overrides to varbinary.